compiler/cpp/src/thrift/generate/t_py_generator.cc

```diff
@@ -2839,4 +2927,5 @@ THRIFT_REGISTER_GENERATOR(
     "                     Package prefix for generated files.\n"
     "    old_style:       Deprecated. Generate old-style classes.\n"
     "    enum:            Generates Python's IntEnum, connects thrift to python enums. Python 3.4 and higher.\n"
+    "    type_hints:        Generate type hints and type checks in write method.\n"
```

Comment:
nit: can you make the spacing line up?
```suggestion
    "    type_hints:      Generate type hints and type checks in write method.\n"
```